### DISCUSSION TITLE
Error message "This chip is ESP32-S2 not ESP32. Wrong --chip argument?"

### DISCUSSION THREAD

**Discussion by Boldfor** [tbnobody/OpenDTU#1049] [Q&A] [answered]
Quick question: I'm smoothly running OpenDTU on an ESP32. Thanks to everyone who contributed to this project. Superb work! :)

When trying to install OpenDTU on an ESP8266 I got the error message `This chip is ESP8266 not ESP32. Wrong --chip argument?`, so I read on the net that OpenDTU does not run on an ESP8266.

However, when trying to install OpenDTU on an ESP32-S2, I get a similar message: `This chip is ESP32-S2 not ESP32. Wrong --chip argument?`

Does this mean that OpenDTU does not run on an ESP32-S2?

I couldn't find anything on the net, hence the question.

**Comment by tbnobody** [comment, marked as answer]
Currently the ESP32-S2 is not officially supported. You can try to generate a environment and compile it yourself but the hardware resources are more limited (e.g. only one core etc.)

**Reply by Boldfor** [reply]
Thanks. Went for a standard ESP32. Works perfectly well.
